JS/script.js

```diff
@@ -1,49 +1,17 @@
-const button1 = document.getElementById('6')
-
-button1.addEventListener('click', () => {
-    const cont1=document.getElementById('71')
-    cont1.style.display='block'
-});
-const button2 = document.getElementById('71')
-
-button2.addEventListener('click', () => {
-    const cont1=document.getElementById('81')
-    cont1.style.display='block'
-});
-const button3 = document.getElementById('81')
-
-button3.addEventListener('click', () => {
-    const cont1=document.getElementById('91')
-    cont1.style.display='block'
-});
-const button4 = document.getElementById('91')
-
-button4.addEventListener('click', () => {
-    const cont1=document.getElementById('101')
-    cont1.style.display='block'
-});
-let button5 = document.getElementById('101')
-
-button5.addEventListener('click', () => {
-    const cont1=document.getElementById('111')
-    cont1.style.display='block'
-});
-let button6 = document.getElementById('12')
 
+for (let i=5;i<14;i++) {
+    const buttonID =document.getElementById(i)
+    buttonID.addEventListener('click', () => {
+        const containerId =document.getElementById(i+1)
+        containerId.style.display='block'
+    })
+}
+const button6 = document.getElementById('15')
 button6.addEventListener('click', () => {
-    const cont1=document.getElementById('71')
-    const cont2=document.getElementById('81')
-    const cont3=document.getElementById('91')
-    const cont4=document.getElementById('101')
-    const cont5=document.getElementById('111')
-    
-    
-
-    
-    cont1.style.display='none'
-    cont2.style.display='none'
-    cont3.style.display='none'
-    cont4.style.display='none'
-    cont5.style.display='none'
-
+    for (let i=6;i<15;i) {

```

Comment:
looks like you missed "i" increment here. Please thing about another elegant approach here to avoid increment on the line 15

Reply:
i wrothe i=i+2 , because each div has a even id number